### DISCUSSION TITLE
How to mark value on the axis???

### DISCUSSION THREAD

**Discussion by Jungle92** [jfree/jfreechart#294] [Q&A]
i want to **mark some value** on the axis .just like this:

<img width="564" alt="a" src="https://user-images.githubusercontent.com/29728121/164195035-d1c7b165-c83e-450a-9de5-d7427e611231.png">

 but i can not find any API to resolve it.

**Comment by trashgod** [comment]
Be sure to update this discussion thread with any new thoughts based on you review of ideas suggested in issue #293. Additional discussion can be found in the old [forum](https://www.jfree.org/forum/viewforum.php?f=3), for [example](https://www.jfree.org/forum/viewtopic.php?t=24865); more related threads are archived [here](https://www.google.com/search?q=site%3Awww.jfree.org%2Fforum+axis+marker+annotation).

**Comment by tracylynne99** [comment]
Use a ValueMarker
